vulnerability in react-server-test-pages

I doubt it'll ever affect anyone, but its an example and nsp is a good practice.

```
dougwade packages/react-server-test-pages ‹master*› » npm install
(node:75611) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.

> react-server-test-pages@0.2.0 prepublish /Users/doug.wade/code/react-server/packages/react-server-test-pages
> gulp build

[23:36:13] Requiring external module babel-register
(node:75612) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
[23:36:13] Using gulpfile ~/code/react-server/packages/react-server-test-pages/gulpfile.babel.js
[23:36:14] Starting 'eslint'...
[23:36:14] Starting 'nsp'...
[23:36:16] 'nsp' errored after 1.43 s
[23:36:16] Error in plugin 'gulp-nsp'
Message:
    (+) 1 vulnerabilities found
┌───────────────┬─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│               │ Insecure Defaults Allow MITM Over TLS                                                                                                                   │
├───────────────┼─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Name          │ engine.io-client                                                                                                                                        │
├───────────────┼─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Installed     │ 1.6.9                                                                                                                                                   │
├───────────────┼─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Vulnerable    │ <= 99.999.99999 || <= 1.6.8                                                                                                                             │
├───────────────┼─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Patched       │ None                                                                                                                                                    │
├───────────────┼─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Path          │ react-server-test-pages@0.2.0 > react-server-cli@0.2.0 > webpack-dev-server@1.12.1 > socket.io@1.4.6 > socket.io-client@1.4.6 > engine.io-client@1.6.9  │
├───────────────┼─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ More Info     │ https://nodesecurity.io/advisories/99                                                                                                                   │
└───────────────┴─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘

[23:36:16] Finished 'eslint' after 2.33 s

npm WARN react-server-test-pages@0.2.0 No repository field.
npm ERR! Darwin 15.4.0
npm ERR! argv "/usr/local/bin/iojs" "/usr/local/bin/npm" "install"
npm ERR! node v6.0.0
npm ERR! npm  v3.8.6
npm ERR! code ELIFECYCLE
npm ERR! react-server-test-pages@0.2.0 prepublish: `gulp build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-server-test-pages@0.2.0 prepublish script 'gulp build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-server-test-pages package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     gulp build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-server-test-pages
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-server-test-pages
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/doug.wade/code/react-server/packages/react-server-test-pages/npm-debug.log
```

--- a/packages/react-server-cli/.nsprc
+++ b/packages/react-server-cli/.nsprc
@@ -1,0 +1,3 @@
+{
+  "exceptions": ["https://nodesecurity.io/advisories/99"]
+}

--- a/packages/react-server-test-pages/.nsprc
+++ b/packages/react-server-test-pages/.nsprc
@@ -1,0 +1,3 @@
+{
+  "exceptions": ["https://nodesecurity.io/advisories/99"]
+}

